src/ConditionBuilder.tsx

```diff
@@ -1,35 +1,30 @@
-import { ConditionSet, Condition, Conditions } from "@nucypher/nucypher-ts";
+import { conditions } from "@nucypher/nucypher-ts";
 import React, { useState } from "react";
-import { Goerli, useEthers } from "@usedapp/core";
+import { Mumbai, useEthers } from "@usedapp/core";
 
 interface Props {
-  conditions?: ConditionSet;
-  setConditions: (value: ConditionSet) => void;
+  conditionExpr?: conditions.ConditionExpression;
+  setConditions: (value: conditions.ConditionExpression) => void;
   enabled: boolean;
 }
 
 export const ConditionBuilder = ({
-  conditions,
+  conditionExpr,
   setConditions,
-  enabled
+  enabled,
 }: Props) => {
   const { library } = useEthers();
-  const NFTBalanceConfig = {
-    contractAddress: '0x932Ca55B9Ef0b3094E8Fa82435b3b4c50d713043', // https://goerli-nfts.vercel.app/
-    standardContractType: 'ERC721',
-    chain: Goerli.chainId,
-    method: 'ownerOf',
-    parameters: [118],
+  const RpcCondition = {
+    chain: Mumbai.chainId,
+    method: "eth_getBalance",
+    parameters: [":userAddress"],
     returnValueTest: {
-      comparator: '==',
-      value: ':userAddress',
+      comparator: ">",
+      value: "0",
```

Comment:
Even though we use a string here, presumably a number is allowed?

Reply:
Yes, both should work fine.